website/cue/reference/components/base/sources.cue

```diff
@@ -1,60 +1,90 @@
 package metadata
 
-base: components: sources: configuration: proxy: {
-	description: """
-		Proxy configuration.
-
-		Configure to proxy traffic through an HTTP(S) proxy when making external requests.
-
-		Similar to common proxy configuration convention, you can set different proxies
-		to use based on the type of traffic being proxied, as well as set specific hosts that
-		should not be proxied.
-		"""
-	required: false
-	type: object: options: {
-		enabled: {
-			description: "Enables proxying support."
-			required:    false
-			type: bool: default: true
-		}
-		http: {
-			description: """
-				Proxy endpoint to use when proxying HTTP traffic.
+base: components: sources: configuration: {
+	graph: {
+		description: """
+			Extra graph configuration
 
-				Must be a valid URI string.
-				"""
-			required: false
-			type: string: examples: ["http://foo.bar:3128"]
-		}
-		https: {
+			Configure output for component when generated with graph command
+			"""
+		required: false
+		type: object: options: node_attributes: {
 			description: """
-				Proxy endpoint to use when proxying HTTPS traffic.
+				Node attributes to add to this component's node in resulting graph
 
-				Must be a valid URI string.
+				They are added to the node as provided
 				"""
 			required: false
-			type: string: examples: ["http://foo.bar:3128"]
+			type: object: {
+				examples: [{
+					color: "red"
+					name:  "Example Node"
+					width: "5.0"
+				}]
+				options: "*": {
+					description: "A single graph node attribute in graphviz DOT language."
+					required:    true
+					type: string: {}
+				}
+			}
 		}
-		no_proxy: {
-			description: """
-				A list of hosts to avoid proxying.
+	}
+	proxy: {
+		description: """
+			Proxy configuration.
 
-				Multiple patterns are allowed:
+			Configure to proxy traffic through an HTTP(S) proxy when making external requests.
 
-				| Pattern             | Example match                                                               |
-				| ------------------- | --------------------------------------------------------------------------- |
-				| Domain names        | `example.com` matches requests to `example.com`                     |
-				| Wildcard domains    | `.example.com` matches requests to `example.com` and its subdomains |
-				| IP addresses        | `127.0.0.1` matches requests to `127.0.0.1`                         |
-				| [CIDR][cidr] blocks | `192.168.0.0/16` matches requests to any IP addresses in this range     |
-				| Splat               | `*` matches all hosts                                                   |
+			Similar to common proxy configuration convention, you can set different proxies
+			to use based on the type of traffic being proxied, as well as set specific hosts that
+			should not be proxied.
```

Comment:
```suggestion
			Similar to common proxy configuration convention, you can set different proxies
			to use based on the type of traffic being proxied. You can also set specific hosts that
			should not be proxied.
```

Reply:
I am not sure how this ended up here. It seems that changes in proxy config were not updated in docs. I will update the proposed changes though, since it already ended up in the PR somehow :smile: